test/setup/setup.ts

```diff
@@ -19,13 +19,17 @@ function resetEnvVars() {
 
 declare global {
 	let jiraHost: string;
+	let jira2Host: string;
```

Comment:
Please don't put this here as it's only specific to one test suite.  Please move it to the test suite file